src/main/java/wolox/training/models/User.java

```diff
@@ -57,10 +70,11 @@
     @Setter(AccessLevel.PRIVATE)
     private Set<Book> books = new HashSet<Book>();
 
-    public User(String name, String username, LocalDate birthDate){
+    public User(String name, String username, LocalDate birthDate, String encodedPassword) {
         setUsername(username);
         setName(name);
         setBirthDate(birthDate);
+        setPassword(encodedPassword);
```

Comment:
This way you have to trust that everyone who uses this method will use an encoded password to get here, but we can assure that it will be that way
I think that the best way to do this is to use the encoder inside the password setter, so you can assure that every password will be encoded before you set it to the object :) What do you think?